### ISSUE TITLE
Why is it special to use the infer_numersenseQA_t5.py for NumersenseQA?

### ISSUE THREAD

**Issue by 2004learner** [liujch1998/GKP#11]
First, it seems that using infer_numersenseQA_t5.py for NumbersenseQA is quite unique. Moreover, in this file, the score computation relies on using the logits, which I find confusing. Why is only the logit of the first token computed? Since words like "no", "one", "two", "three", etc., are represented by a single token, it seems strangeto focus solely on the first token's logit. Also, if the score is being calculated through loss, wouldn't it make sense to merge the functionality of infer_numersenseQA_t5.py with infer_t5.py?
